eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
       - task: PowerShell@2
         displayName: Prep template pipeline for release
-        condition: and(succeeded(),eq(variables['TestPipeline'],'true'))
+        condition: and(succeeded(),eq('${{ parameters.TestPipeline }}','true'))
```

Comment:
Lets switch to a template condition.

Reply:
@weshaggard I'm curious, what's your thinking on condition vs. template condition in general? In my mind trying to use as little templating syntax as possible makes the yaml easier to comprehend in the long run. In this case I can see how it's a little more obvious to use the template syntax as opposed to nesting logic within the `and` one-liner. Are there other things I should be considering?